pages/index.js

```diff
@@ -56,10 +56,10 @@ export default function Home() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Powered by{' '}
+          Powered by{" "}
           <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
```

Comment:
@ricardoav777 Pls, change the description here -> Made with ❤️  by Coderiosity team